contracts/SwapFlashLoan.sol

```diff
@@ -0,0 +1,169 @@
+// SPDX-License-Identifier: MIT WITH AGPL-3.0-only
```

Comment:
we should probably use just `MIT` to be consistent with the other contracts

also might be worth setting up [hardhat-spdx-license-identifier](https://www.npmjs.com/package/hardhat-spdx-license-identifier) to automate this

Reply:
Added to backlog https://github.com/saddle-finance/saddle-contract/issues/313